modules/installing-ocp-agent-boot.adoc

```diff
@@ -19,4 +19,32 @@ $ openshift-install --dir <install_directory> agent create image
 +
 NOTE: Red Hat Enterprise Linux CoreOS (RHCOS) supports multipathing on the primary disk, allowing stronger resilience to hardware failure to achieve higher host availability. Multipathing is enabled by default in the agent ISO image, with a default `/etc/multipath.conf` configuration.
 
-. Boot the `agent.x86_64.iso` or `agent.aarch64.iso` image on the bare metal machines.
+. Boot the `agent.x86_64.iso`, `agent.aarch64.iso` or `agent.s390x.iso` image on the bare metal machines.
+
+== Adding {ibm-z-name} agents with {op-system-base} KVM for ISO boot
+
+Use the following procedure to manually add {ibm-z-name} agents with {op-system-base} KVM for ISO boot.
+
+.Procedure
+
+. Boot your {op-system-base} KVM machine.
+
+. To deploy the virtual server, run the `virt-install` command with the following parameters:
++
+[source,terminal]
+----
+$ virt-install     
+    --name <vm_name> \ 
+    --autostart \   
+    --memory=<memory> \  
+    --cpu host \    
+    --vcpus=<vcpus> \
+    --cdrom <agent.iso_image> \ <1>  
+    --disk pool=default,size=<disk_pool_size> \ 
+    --network network:default,mac=<mac_address> \   
+    --graphics none \    
+    --noautoconsole \     
+    --os-variant rhel9.0 \ 
+    --wait=-1
+----
+<1> For the `--cdrom` parameter, specify the location of the ISO image on the HTTP or HTTPS server.
```

Comment:
This can be removed you have that in the `installing-ocp-agent-ibm-z-kvm.adoc` file now

```suggestion

```

Reply:
+1 this also needs to be removed now that you've included the ifdef logic in the other module